docs/tasks.md

```diff
@@ -224,3 +224,25 @@ The message to show.
 ```
 echo: Hello world!
 ```
+
+## retry
+
+`retry` task retries the job from a failed task automatically.
+
+**Option**
+
+`count` option and optional `sleep_time` option.
+`count` option is required.
+
+The count option is the number of attempts to execute each task.
+The sleep time option is the number of seconds in an interval of retries.
+
+**Example**
+
+```
+retry: count=3 sleep_time=30
+  execute: echo 1
+  execute: test -e /tmp/foo.txt
```

Comment:
(question) is there any reason written like this rather than:

```ruby
retry:
  count: 3
  sleep_time:3
execute: echo 1
execute: test -e /tmp/foo.txt
```

Why I'm asking, a task definition supports yml syntax, hence 'key=value' style options look weird ;)